src/mb_enhanced_cover_art_uploads/providers/base.ts

```diff
@@ -116,6 +116,7 @@ export abstract class CoverArtProvider {
 export interface ParsedTrackImage {
     url: string;
     trackNumber?: string;
+    customComment?: [string, string];  // singular, plural
```

Comment:
```suggestion
    customCommentPrefix?: [string, string];  // singular, plural
```
Makes it clearer whether we expect a comment prefix or a full comment which will be used verbatim.

Reply:
Good point. I'm sure you'll find loads of other poorly-named variables/properties in the changes over the last couple of months 😅